### DISCUSSION TITLE
thermia itec eco 12 vs. Home Assistant

### DISCUSSION THREAD

**Discussion by wes-4444** [klejejs/ha-thermia-heat-pump-integration#98] [Q&A]
Hi here, I'm kind of a noob but would very much like to be able to integrate my heatpump (Thermia Itec Eco 12, connected via Thermia Online) with home assistant in order for it to work together with my solar panels.  I'm already stuck on adding the integration. When looking for "thermia", nothing comes up.

Any help is much appreciated.

**Comment by klejejs** [comment]
Hey @wes-4444! Did you download it using HACS or manually? Do you have `thermia` folder present under `custom_components` in your `config` directory?

**Comment by wes-4444** [comment]
Hi Klejejs, ages later but in the meanwhile a bit more _knowledgeable_ .  Initially I tried on an HA instance that runs on my Velbus Signum. As this is a custom integration with very limited hardware, it just wasn't possible to install and run the integration. In the meanwhile I installed a separate HA server and also your thermia integration -thanks for that!  I see the different sensors, however 'warm water boost' is not there. Does this require a change on the heatpump itself?  Thanks very much for your help.
